### PR TITLE
Comment out mdns search in Android to isolate possible low memory causes

### PR DESCRIPTION
**Pull request by ziransun**
Jira-Issue: WP-406

### CODE REVIEW

**Comment by habibvirji** [comment]
Apologise Ziran , can you resubmit this to latest master

File: webinos/pzp/lib/pzp_sessionHandling.js [hunk 1]
```diff
@@ -405,12 +405,12 @@ Pzp.prototype.connect = function (conn_key, conn_csr, code, address, callback) {
                 //get pzp TLS port number    
                 var pzp_tlsServer = 8040;
                 var filename;
-  		        if((os.type().toLowerCase() == "linux") && (os.platform().toLowerCase() == "android"))
+  		       /* if((os.type().toLowerCase() == "linux") && (os.platform().toLowerCase() == "android"))
                 {
                   //TODO: the full file path has to be given - check if any alternative way
                   filename = "/data/data/org.webinos.app/node_modules/webinos/wp4/webinos_config.json";
```
Comment (andreasbotsikas) [inline webinos/pzp/lib/pzp_sessionHandling.js:411]: We have just made a change in pzp using path.resolve(__dirname,"../../somewhere/in/the/source.js") and it works on aNode

**Comment by andreasbotsikas** [comment]
Ziran do you think that we need this, cause you must rebase it...

**Comment by habibvirji** [comment]
@ziransun should i close this PR

**Pull request closed without merging**